x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/action/EsqlQueryResponseTests.java

```diff
@@ -236,7 +236,7 @@ private Page randomPage(List<ColumnInfoImpl> columns) {
     protected EsqlQueryResponse mutateInstance(EsqlQueryResponse instance) {
         boolean allNull = true;
         for (ColumnInfoImpl info : instance.columns()) {
-            if (info.type() != DataType.NULL) {
+            if (info.type() != DataType.NULL && info.type() != DataType.UNSUPPORTED) {
```

Comment:
Might be worth adding a comment here that for UNSUPPORTED the `null` value is used throughout the test suite, thus the check here.